### DISCUSSION TITLE
Creator Subscription

### DISCUSSION THREAD

**Discussion by ronaldroe** [dailydotdev/daily#997] [Feature Requests]
I looked all over the UI to see if I was just missing it, but is there functionality to allow subscribing to creators on here? There's a few whose articles I want to see more prevalent in the feed.

Awesome platform, btw. Thank you.

**Comment by idoshamun** [comment]
Hi @ronaldroe,

We don't support this feature at the moment 😢
Thank you for suggesting :)
